src/pages/daily-runs/[date]/[time].tsx

```diff
@@ -98,7 +112,10 @@ const DailyRun: NextPage = () => {
           </div>
         </div>
       )}
-      <BreakoutsList data={breakoutsData} />
+      <BreakoutsList
+        data={breakoutsData}
+        disableBuy={!isDateToday(dateString)}
```

Comment:
Is it okay sending in disableBuy props like this instead of including it in the props data object?

Reply:
Yes, this is totally fine.
It is actually preferred not to make the Listing component handling logics af various complexity - it should as simply as possibly just list data as is.